shipit.rubygems.yml

```diff
@@ -1 +1,3 @@
 # using the default shipit config
+ hide:
+  -policial
```

Comment:
Looks right, based on the docs: https://github.com/Shopify/shipit-engine#ci

Noob question: Does it matter if there isn't a space between the `-` and `policial`?

Reply:
I think it probably needs to be there! I'll add